packages/editor/src/components/post-template/block-theme.js

```diff
@@ -58,25 +66,15 @@ export default function BlockThemeControl( { id } ) {
 				<>
 					<MenuGroup>
 						<MenuItem
-							onClick={ () => {
-								setRenderingMode( 'template-only' );
+							onClick={ ( event ) => {
+								editTemplate.onClick( event );
 								onClose();
 								createSuccessNotice(
 									__(
 										'Editing template. Changes made here affect all posts and pages that use the template.'
 									),
 									{
 										type: 'snackbar',
-										actions: [
-											{
-												label: __( 'Go back' ),
-												onClick: () =>
-													setRenderingMode(
-														getEditorSettings()
-															.defaultRenderingMode
-													),
-											},
-										],
```

Comment:
Should this be deleted?

Reply:
With the current setup of getPostLinkProps we don't have  `goBack` set at this point as we don't have history yet as we haven't moved to the new entity. I wondered if it was actually needed as we are not providing this back link in a notice for pattern editing, but was planning to circle back and reconsider this.  I now think I have a plan for providing the goBack option prior to having history, so will add it back in if that plan works out.